litmus-portal/frontend/src/utils/yamlUtils.ts

```diff
@@ -32,7 +32,7 @@ const parsed = (yaml: string) => {
     const parsedYaml = YAML.parse(file as string);
     try {
       const embeddedYaml =
-        parsedYaml.spec.workflowSpec.templates[1].inputs.artifacts[0].raw.data;
+        parsedYaml.spec.templates[1].inputs.artifacts[0].raw.data;
```

Comment:
@ishangupta-ds It is changed for parsing new yamls??

Reply:
We will be using the old yaml template (kind: Workflow) not new one (kind: CronWorkflow) for schedule now option for all the workflows. cc: @ksatchit